Build/Strategic/Meanwhile.cc

```diff
@@ -438,6 +442,7 @@ static void BeginMeanwhileCallBack(MessageBoxReturnValue const bExitValue)
 	else
 	{
 		// skipped scene!
+		gfInMeanwhile = FALSE;
```

Comment:
Saving while in a cutscene ("Meanwhile") causes errors in the save, so I check against it to avoid saving.
gfInMeanwhile is true when the game is in a cutscene. It is not set back to false if the player skips a cutscene. Thus the game always thinks it is in a cutscene, and would never save if a player skips one.

This is basically a vanilla error that had no effect previously, as the variable is never used alone.